### PR TITLE
Use nearlyEquals for estimate comparison

### PR DESCRIPTION
**Pull request by kokosing**
Use nearlyEquals for estimate comparison

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/util/MoreMath.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+    /**
+     * See http://floating-point-gui.de/errors/comparison/
+     */
+    public static boolean nearlyEqual(double a, double b, double epsilon)
+    {
+        final double absA = Math.abs(a);
+        final double absB = Math.abs(b);
+        final double diff = Math.abs(a - b);
```
Comment (findepi) [inline presto-main/src/main/java/com/facebook/presto/util/MoreMath.java:29]: If the code convention applies to copy&pasted code too, remove `final`

File: presto-main/src/main/java/com/facebook/presto/util/MoreMath.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+    /**
+     * See http://floating-point-gui.de/errors/comparison/
+     */
+    public static boolean nearlyEqual(float a, float b, float epsilon)
+    {
+        final float absA = Math.abs(a);
+        final float absB = Math.abs(b);
+        final float diff = Math.abs(a - b);
```
Comment (findepi) [inline presto-main/src/main/java/com/facebook/presto/util/MoreMath.java:51]: If the code convention applies to copy&pasted code too, remove final

File: presto-main/src/test/java/com/facebook/presto/cost/EstimateAssertion.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+{
+    private static double TOLERANCE = 0.0000001;
+
+    private EstimateAssertion()
+    {
+    }
+
+    public static void assertEstimateEquals(double actual, double expected, String messageFormat, Object... mesaggeObjects)
+    {
+        assertEquals(MoreMath.nearlyEqual(actual, expected, TOLERANCE), String.format(messageFormat, mesaggeObjects));
```
Comment (findepi) [inline presto-main/src/test/java/com/facebook/presto/cost/EstimateAssertion.java]: static import format

**Comment by losipiuk** [comment]
Looks good modulo verbal comments

File: presto-main/src/test/java/com/facebook/presto/cost/PlanNodeStatsAssertion.java [hunk 4]
```diff
@@ -78,18 +79,20 @@ public PlanNodeStatsAssertion equalTo(PlanNodeStatsEstimate expected)
     {
         assertEquals(actual.getOutputRowCount(), expected.getOutputRowCount());
 
-        for (Symbol symbol : union(expected.getSymbolStatistics().keySet(), actual.getSymbolStatistics().keySet())) {
+        for (Symbol symbol : union(expected.getSymbolStatistics()
```
Comment (findepi) [inline presto-main/src/test/java/com/facebook/presto/cost/PlanNodeStatsAssertion.java:82]: unrelated change

**Review by findepi — CHANGES_REQUESTED** [review]
